package/harvester-os/files/system/oem/06_ntp.yaml

```diff
@@ -0,0 +1,7 @@
+name: "Enable Systemd Time Wait Sync"
+stages:
+  initramfs:
+    - name: "enable systemd-time-wait-sync.service"
```

Comment:
If the user doesn't specify NTP servers, services still need to wait 3 minutes to start.
Could we detect if the user set NTP servers first and then overridden systemd units to avoid the wait?

Reply:
Let me check about this one.

Reply:
I enable time sync wait service if and only NTP server config is existing.